turborepo-tests/integration/tests/_fixtures/single_package/package.json

```
@@ -1,6 +1,7 @@
 {
   "name": "my-pkg",
   "scripts": {
-    "build": "echo 'building' > foo"
+    "build": "echo 'building' > foo",
+    "test": "echo 'testing'"
```

Can we change this to `cat foo`? The `[[ ( -f foo ) && $(cat foo) == 'building' ]]` script depended on the outputs of `build` being present. From reading through the changes I don't think we ended up ever testing output restoration using this, but I don't think it can hurt to keep that aspect of test task.
```suggestion
    "test": "cat foo"
```

---

good idea, done